processor/src/main/kotlin/permissions/dispatcher/processor/util/Helpers.kt

```diff
@@ -48,7 +48,7 @@ fun <A : Annotation> findMatchingMethodForNeeds(needsElement: ExecutableElement,
 fun varargsParametersCodeBlock(needsElement: ExecutableElement): CodeBlock {
     val varargsCall = CodeBlock.builder()
     needsElement.parameters.forEachIndexed { i, it ->
-        varargsCall.add("\$L", it.simpleString())
```

Comment:
In JavaPoet this writing style is appropriate. This is actually super misleading though

Reply:
Yes